engine/src/core/utils/time/time.h

```diff
@@ -0,0 +1,60 @@
+//
```

Comment:
All our methods start with capitals letters. Also, add documentation and see if you can use this class in already existing Timer class to have it all clean and tidy.